packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +57,266 @@ export default {
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * To allow multiple options to be selected, use the `multiple` attribute. It’s a good practice to use `clearable` when this option is enabled. To use the checkbox with tags variant, set the `useTags` variant to `true`.  To set multiple values at once, set value to a space-delimited list of values.  The preferred placement of the select’s listbox can be set with the `placement` attribute. Note that the actual position may vary to ensure the panel remains in the viewport. Valid placements are `top` and `bottom`.
+ * Use the `placement` attribute to define where the select panel should appear.
  */
 
-export const DisabledMultiple = {
-  name: 'Disabled x Multiple',
-  parameters: {
-    controls: {
-      exclude: ['open']
-    }
-  },
-  render: (args: { 'open-attr'?: string }) => {
-    delete args['open-attr'];
-
-    return html`<div class="h-[340px] w-full">
-      ${generateTemplate({
-        options: {
-          classes: 'w-full [&>tbody>tr>td]:w-[50%]'
-        },
-        axis: {
-          y: {
-            type: 'attribute',
-            name: 'useTags',
-            values: [false, true]
-          },
-          x: {
-            type: 'attribute',
-            name: 'disabled',
-            values: [false, true]
-          }
-        },
-        constants: [
-          clearableConstant,
-          multipleConstant,
-          fiveOptionsConstant,
-          {
-            type: 'attribute',
-            name: 'value',
-            value: 'option-1 option-2 option-3 option-4'
-          }
-        ],
-        args
-      })}
-    </div>`;
-  }
+export const Placement = {
+  render: () => html`
+    <div class="flex items-center gap-12 h-[500px]">
+      <sd-select class="self-baseline" size="lg" label="Label" placement="bottom" value="">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+
+      <sd-select size="lg" label="Label" placement="top" value="">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * `sd-select` with valid and invalid styles.
+ * Use the `label` slot to add a label with custom markup.
```

Comment:
Use the "label" attribute to provide an accessible label for the select element.
Use the "label" slot to add custom markup.

Reply:
Updated